test/edu-benefits/pages/applicantInformation.unit.spec.jsx

```diff
@@ -44,12 +42,8 @@ function fillInformation(find) {
   });
 }
 
-describe('Basic applicantInformation', () => {
-  const applicantSchema = {
-    definitions: _.pick(['fullName', 'ssn', 'date', 'gender', 'relationship'],
-                        fullSchema1990e.definitions)
-  };
-  const { schema, uiSchema } = applicantInformation(applicantSchema);
+describe('Edu basic applicantInformation', () => {
+  const { schema, uiSchema } = applicantInformation(fullSchema1990e);
```

Comment:
There's a test in the 1990n folder that does that (though could be more extensive).